internal/console/console.go

```diff
@@ -0,0 +1,17 @@
+package console
+
+import (
+	"fmt"
+)
+
+// Printf defines an (overridable) function for printing to the console.
+var Printf = func(format string, a ...any) {
```

Comment:
There's no way to just override fmt.Printf? This seems like something that should be upstreamed in whatever wasm tooling we're using.

Reply:
Not as far as I know and when I tried to use a pipe to collect stdout and stderr, it did not work in WASM land